src/components/SuggestionTags/Tags/PriceImpactWarningSuggestion.tsx

```diff
@@ -0,0 +1,200 @@
+
+interface Props {
+  isWarning: boolean;
+  isDanger: boolean;
+  birdeyeRate: number | null;
+  isHighPriceImpact: boolean;
+  priceDifferencePct: number;
+  fromTokenInfo: TokenInfo;
+  toTokenInfo: TokenInfo;
+  quoteResponse: QuoteResponse;
+}
+
+const PriceImpactWarningSuggestion = ({
+  birdeyeRate,
+  isWarning,
+  isDanger,
+  isHighPriceImpact,
+  priceDifferencePct,
+  fromTokenInfo,
+  toTokenInfo,
+  quoteResponse,
+}: Props) => {
+  const { tokenPriceMap } = useUSDValue();
+  const { inAmount, outAmount, priceImpactPct } = quoteResponse;
+
+  const tradeValue = useMemo(
+    () => new Decimal(inAmount.toString()).div(10 ** fromTokenInfo.decimals),
+    [inAmount, fromTokenInfo.decimals],
+  );
+
+  const fromTokenValue = useMemo(
+    () => tradeValue.mul(tokenPriceMap[fromTokenInfo.address || '']?.usd || 0),
+    [tradeValue, fromTokenInfo.address, tokenPriceMap],
+  );
+
+  const toTokenValue = useMemo(
+    () =>
+      new Decimal(outAmount.toString())
+        .div(10 ** toTokenInfo.decimals)
+        .mul(tokenPriceMap[toTokenInfo.address || '']?.usd || 0),
+    [outAmount, toTokenInfo.address, toTokenInfo.decimals, tokenPriceMap],
+  );
+
+  const shouldPromptDca = useMemo(() => fromTokenValue.gte(1_000), [fromTokenValue]);
+
+  const [reverse, setReverse] = React.useState(true);
+
+  const onReverse: React.MouseEventHandler = React.useCallback((event) => {
+    event.stopPropagation();
+    setReverse((prevState) => !prevState);
+  }, []);
+
+  const swapRouteInfo = useSwapRouteInfo();
+  const renderWarningColor = useMemo(
+    () => isWarning || swapRouteInfo?.isMediumImpact,
+    [isWarning, swapRouteInfo?.isMediumImpact],
+  );
+
+  const renderDangerColor = useMemo(
+    () => isDanger || swapRouteInfo?.isHighImpact,
+    [isDanger, swapRouteInfo?.isHighImpact],
+  );
+
+  const isMobile = useMobile();
+
+  const { pillTitle, title, description } = useMemo(() => {
+    if (isHighPriceImpact) {
+      return {
+        pillTitle: `${new Decimal(priceImpactPct).mul(100).toNumber().toFixed(2)}% Price Impac`,
+        title: shouldPromptDca ? `Looks like your trade size is too large.` : `Looks like you are getting a bad rate.`,
+        description: shouldPromptDca ? (
+          <Link
+            shallow
+            href={`/dca/${fromTokenInfo.address}-${toTokenInfo.address}?${new URLSearchParams({
+              inAmount: tradeValue.toString(),
+              frequency: 'minute',
+              frequencyValue: '10',
+            }).toString()}`}
+            className="text-xs cursor-pointer flex items-center justify-center mb-2"
+          >
+            <JupButton size="sm" type="button" className="!w-[120px]">
+              <div className="flex gap-x-2 items-center justify-center">
+                <span>{`Go to DCA`}</span>
+                <RightArrowIcon />
+              </div>
+            </JupButton>
+          </Link>
+        ) : (
+          `Please refresh quote or trade with caution.`
```

Comment:
Need to add the translations macro here? Seems like the usage in the div below never wrap translations either.